desktop/mac/index.md

```diff
@@ -174,19 +174,17 @@ Select **Use the new Virtualization framework** to allow Docker Desktop to use t
 
 #### Enable VirtioFS
 
-Select the **Enable VirtioFS accelerated directory** sharing option to enable VirtioFS. You must install Docker Desktop 4.6 or higher and macOS 12.2 to use VirtioFS.
+The 4.6 release of Docker Desktop for Mac lets developers use a new experimental file-sharing implementation called [virtioFS](https://virtio-fs.gitlab.io/){: target='_blank' rel='noopener' class='_'}; the current default is gRPC-FUSE. During testing, virtioFS has reduced the time taken to sync changes between the host and VM up to 90%. For more details, see our < need link for 4.6 blog post>.
```

Comment:
```suggestion
The 4.6 release of Docker Desktop for Mac lets developers use a new experimental file-sharing implementation called [virtiofs](https://virtio-fs.gitlab.io/){: target='_blank' rel='noopener' class='_'}; the current default is gRPC-FUSE. During testing, virtioFS has reduced the time taken to sync changes between the host and VM up to 90%. For more details, see our < need link for 4.6 blog post>.
```

Reply:
I'd suggest not to call out the version number in the docs as a) they can find the release when a specific feature was introduced in the release notes and b) when we start supporting the feature in future releases, we'll have to keep adding the version numbers to reflect this change.

Reply:
good idea, thanks @usha-mandya